MediaBrowser.MediaEncoding/Probing/ProbeResultNormalizer.cs

```diff
@@ -494,42 +495,44 @@ private List<NameValuePair> ReadValueArray(XmlReader reader)
         private void ProcessPairs(string key, List<NameValuePair> pairs, MediaInfo info)
         {
             IList<BaseItemPerson> peoples = new List<BaseItemPerson>();
+            var distinctPairs = pairs.Select(p => p.Value)
```

Comment:
Do the trim and distincting (in that order!) for all of these people lists, not just studios